src/User32.Desktop/User32.cs

```diff
@@ -144,6 +144,17 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern unsafe IntPtr SendMessage(IntPtr hWnd, int wMsg, void* wParam, void* lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern unsafe IntPtr SendMessage(IntPtr hWnd, WindowMessage wMsg, void* wParam, void* lParam);
```

Comment:
I remember that for other cases you had to make a class instead of an enum to side-step the no-cast problem but yes maybe just having the enum overload would be enough.

Potentially with an helper to create an user message (As they are specified to be between `WM_USER` and `0x7FFF`)